### PR TITLE
core: remove automapper

### PR DESCRIPTION
**Pull request by ngosang**

### CODE REVIEW

File: src/Jackett.Common/Services/CacheService.cs [hunk 1]
```diff
@@ -68,7 +68,7 @@ public void CacheResults(IIndexer indexer, TorznabQuery query, List<ReleaseInfo>
                 var trackerCacheQuery = new TrackerCacheQuery
                 {
                     Created = DateTime.Now,
-                    Results = releases
+                    Results = releases.Select(r => (ReleaseInfo)r.Clone()).ToList()
```
Comment (PublisherApplications) [inline src/Jackett.Common/Services/CacheService.cs:71]: good job

**Review by PublisherApplications — APPROVED** [review]
Good job

File: src/Jackett.Common/Utils/MapperUtil.cs
Comment (PublisherApplications) [inline src/Jackett.Common/Utils/MapperUtil.cs:1]: Why did this file have been deleted?

**Review by PublisherApplications — CHANGES_REQUESTED** [review]
I'm not sure about this deleting file thing...

**Comment by ilike2burnthing** [comment]
Your patronisingly toned 'reviews' are not necessary for year old PRs from one of the maintainers.

**Pull request closed without merging**